sqlbrite-lint/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'kotlin'
+
+dependencies {
+  compileOnly rootProject.ext.kotlinStdLib
```

Comment:
This should be `implementation` since, in theory, future versions of lint could not be based on Kotlin and thus not have the stdlib already on the classpath.

Reply:
If I mark it as `implementation` instead of `compileOnly` I have this error:

```
Execution failed for task :sqlbrite:prepareLintJar.
Found more than one jar in the 'lintChecks' configuration. Only one file is supported. If using a separate Gradle project, make sure compilation dependencies are using compileOnly
```

Reply:
That's... interesting. I mean it makes sense, but I guess we have to assume that lint will always contain the Kotlin stdlib as a result.